packages/application-generic/src/health/queue-health-indicator.service.ts

```diff
@@ -0,0 +1,67 @@
+import {
+  HealthCheckError,
+  HealthIndicator,
+  HealthIndicatorResult,
+} from '@nestjs/terminus';
+import { Injectable, Logger } from '@nestjs/common';
+
+import { QueueBaseService } from '../services';
+import { IHealthIndicator } from './health-indicator.interface';
+
+@Injectable()
+export abstract class QueueHealthIndicator
+  extends HealthIndicator
+  implements IHealthIndicator
+{
+  constructor(
+    private queueBaseService: QueueBaseService,
```

Comment:
```suggestion
    private queueService: QueueBaseService,
```
Just because we are passing other services that satisfy the QueueBaseService interface.